_scripts/checks.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+set -ex
+
+if [[ ! $(which rustup) ]]; then
+  curl https://sh.rustup.rs -sSf | sh -s -- -y
+  export PATH=$PATH:$HOME/.cargo/bin/
+fi
+if [[ $(ubuntu --version) -eq 20.04 ]]; then
+  ubuntu=y
+else
+  ubuntu=n
+fi
```

Comment:
What's `ubuntu` here for?

Reply:
It's of no use. I realized it later after pushing the commits.

Reply:
Will fix.